### DISCUSSION TITLE
How to extend react sketch canvas in a way to make it scale responsively (e.g.: to its parent container)

### DISCUSSION THREAD

**Discussion by mtin79** [vinothpandian/react-sketch-canvas#28] [Q&A] [answered]
Hi out there react sketch users and maintainers. Does anyone know how to set up RSC in a way so that the drawing scales responsively?

**Comment by vinothpandian** [comment, marked as answer]
Hey @mtin79,

You can set the SVG's [viewBox](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/viewBox) property on the canvas Ref DOM node's firstChild to scale responsively
